src/cbc_sdk/platform/alerts.py

```diff
@@ -1415,39 +1428,45 @@ def _update_status(self, status, remediation, comment):
             request["criteria"] = criteria
         if query:
             request["query"] = query
-        if remediation is not None:
-            request["closure_reason"] = remediation
-        if comment is not None:
-            request["note"] = comment
+        if closure_reason is not None:
+            request["closure_reason"] = closure_reason
+        if determination is not None:
+            request["determination"] = determination
+        if note is not None:
+            request["note"] = note
         resp = self._cb.post_object(self._bulkupdate_url.format(self._cb.credentials.org_key), body=request)
         output = resp.json()
-        return output["request_id"]
+        return Job(self._cb, output["request_id"]).await_completion()
```

Comment:
No don't add `.await_completion()` as we don't want to block. The user can decide if they want to call it themselves

Reply:
confused on this. according to the sdk docs await_completion doesn't block. Am I misunderstanding something here? i did find it confusing that await wouldn't block. but yeah I'll pull it out